### DISCUSSION TITLE
Frigate docs

### DISCUSSION THREAD

**Discussion by McPaHa** [blakeblackshear/frigate#2891] [General]
Hi!

Did I notice right that https://docs.frigate.video/configuration/index hasn't been updated yet?

At least I don't find any mentions about retain modes or Stationary objects?

Anyway - Great Work!!!

**Comment by blakeblackshear** [comment]
It's updated in the example config. Were you expecting it somewhere else?

**Comment by McPaHa** [comment]
Ok... checking it... probably I'm just blind ;)
